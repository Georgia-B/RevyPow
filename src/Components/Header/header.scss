.header {
    height: 10vh;
    display: flex;
    align-items: center;
    justify-content: center;

    &__title {
        font-weight: lighter;
        font-size: 34px;
        margin: 0;
        line-height: 1;
    }
}