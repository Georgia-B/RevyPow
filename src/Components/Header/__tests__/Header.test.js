import React from 'react'
import { cleanup, render } from '@testing-library/react'
import Header from '../Header';

describe('Components | <Header />', () => {
    afterEach(cleanup);
    it('should render correctly', () => {
        const component = render(<Header />);
        expect(component.asFragment()).toMatchSnapshot();
    })
})