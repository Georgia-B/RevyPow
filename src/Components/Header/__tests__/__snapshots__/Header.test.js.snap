// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components | <Header /> should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h1
      class="header__title"
    >
      RevyPow
    </h1>
  </div>
</DocumentFragment>
`;
